flower/utils/pg_storage.py

```diff
@@ -18,16 +22,40 @@
     (
         id SERIAL PRIMARY KEY,
         time TIMESTAMP NOT NULL,
-        data JSONB NOT NULL,
+        uuid VARCHAR(36) NOT NULL,
```

Comment:
@johnarnold 36 characters is too few... I've said in a previous comment that these are not always uuids but can be arbitrary strings -- which can easily be longer than 36 characters. As an example, the current maximum length of a task id in our database is 129 characters.

Reply:
@bosonogi Just letting you know I've been slammed with other work lately so I haven't gotten back to this but I will try.  For our environment it seems like our volume is stressful to PostgreSQL, so I may write up a very simple redis provider and contribute into your PR.

Reply:
@johnarnold cool; work situation is similar at my end.

An idea for an alternative approach has been brewing that would sidestep Flower (and its in-memory architecture). As I only use a small subset of what Flower does (start tasks and look up task details using its HTTP API), it seems to me it should be easy to make a Django Rest Framework app that does this -- utilizing the event data we already use/have.